magellan-library/src/main/java/com/wealthfront/magellan/transitions/CrossfadeTransition.kt

```diff
@@ -2,31 +2,58 @@ package com.wealthfront.magellan.transitions
 
 import android.animation.Animator
 import android.animation.AnimatorListenerAdapter
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
+import android.util.Property
 import android.view.View
 import android.view.View.GONE
 import com.wealthfront.magellan.Direction
 
 public class CrossfadeTransition : MagellanTransition {
 
+  private var animationSet: Animator? = null
```

Comment:
the existing CrossfadeTransition worked by fading out, and then fading in sequentially. I'm not sure why that was, but this will do both simultaneously as it's simpler to cancel that

Reply:
err wait a sec. AnimatorSet has a `playSequentially` command. Going to use that